mne/preprocessing/eyetracking/eyetracking.py

```diff
@@ -5,7 +5,100 @@
 
 import numpy as np
 
+from ...io import BaseRaw
 from ...io.constants import FIFF
+from ...utils import logger, _check_preload, _validate_type, warn
+
+
+def interpolate_blinks(raw, buffer=0.05, match="BAD_blink"):
+    """Interpolate eyetracking signals during blinks.
+
+    This function uses the timing of blink annotations to estimate missing
+    data. Operates in place.
+
+    Parameters
+    ----------
+    raw : instance of Raw
+        The raw data with at least one ``'pupil'`` or ``'eyegaze'`` channel.
+    buffer : float | array-like of float, shape ``(2,))``
+        The time in seconds before and after a blink to consider invalid and
+        include in the segment to be interpolated over. Default is ``0.05`` seconds
+        (50 ms). if array-like, the first element is the time before the blink and the
+        second element is the time after the blink to consider invalid, for example,
+        ``(0.025, .1)``.
+    match : str | list of str
+        The description of annotations to interpolate over. If a list, data within
+        annotations matching any of the strings in the list will interpolated over.
```

Comment:
```suggestion
        The description of annotations to interpolate over. If a list, the data within
        all annotations that match any of the strings in the list will be interpolated over.
```
I gave this a small update - as a non-native speaker I found the sentence hard to parse, maybe this is clearer/easier to read?

Reply:
I agree that this reads better, thank you!